pkg/controller/jobs/deployment/deployment_webhook.go

```diff
@@ -76,7 +75,7 @@ var _ admission.CustomValidator = &Webhook{}
 func (wh *Webhook) ValidateCreate(ctx context.Context, obj runtime.Object) (warnings admission.Warnings, err error) {
 	deployment := fromObject(obj)
 
-	log := ctrl.LoggerFrom(ctx).WithName("deployment-webhook").WithValues("deployment", klog.KObj(deployment))
+	log := ctrl.LoggerFrom(ctx).WithName("deployment-webhook")
```

Comment:
Actually, can you show the log line before and after?

Reply:
Before:

```
2024-11-18T16:52:08.405914+02:00      LEVEL(-5)       admission.deployment-webhook    deployment/deployment_webhook.go:80     Validating create       

{
  "webhookGroup": "apps",
  "webhookKind": "Deployment",
  "Deployment": {
    "name": "deployment",
    "namespace": "deployment-2wrmw"
  },
  "namespace": "deployment-2wrmw",
  "name": "deployment",
  "resource": {
    "group": "apps",
    "version": "v1",
    "resource": "deployments"
  },
  "user": "admin",
  "requestID": "1721f814-fd0a-480a-9fd8-af592e173b99",
  "deployment": {
    "name": "deployment",
    "namespace": "deployment-2wrmw"
  }
}

```

After
```
  2024-11-18T16:54:08.321662+02:00      LEVEL(-5)       admission.deployment-webhook    deployment/deployment_webhook.go:98     Validating update       

{
  "webhookGroup": "apps",
  "webhookKind": "Deployment",
  "Deployment": {
    "name": "deployment-with-queue-name",
    "namespace": "deployment-sd7gl"
  },
  "namespace": "deployment-sd7gl",
  "name": "deployment-with-queue-name",
  "resource": {
    "group": "apps",
    "version": "v1",
    "resource": "deployments"
  },
  "user": "admin",
  "requestID": "dadb6d53-0e59-48c3-9405-9f6e43baa16c"
}
```

Reply:
thanks for clarifying